README.md

```diff
@@ -0,0 +1,115 @@
+<a href="https://github.com/hypothesis/h/actions/workflows/ci.yml?query=branch%3Amain"><img src="https://img.shields.io/github/actions/workflow/status/hypothesis/h/ci.yml?branch=main"></a>
```

Comment:
Need to check this `README.md` against the `README.rst` and `HACKING.md` on `main`, make sure nothing's been lost.